canvas_syllabus_export/settings/local.py

```diff
@@ -16,6 +16,9 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
+# Base URL for Canvas instance
+BASE_URL = "https://canvas.harvard.edu/api"
```

Comment:
Note: this is fine for local development (hard-coded). So you can leave this as-is.